### ISSUE TITLE
Principle X number in header?

### ISSUE THREAD

**Issue by jaredwyee** [cagov/design-system#660]
Recommendation: Include the Principle and the numbers in the heading levels. They are currently missing.

![image](https://user-images.githubusercontent.com/85529382/153976762-856b43a3-693f-4ab3-bba0-d16be6c91f58.png)

**WCAG 2.1 Guidelines**
[1.3.1 Info and Relationships (Level A)](https://www.w3.org/TR/WCAG21/#info-and-relationships)

[2.4.6 Headings and Labels (Level AA)](https://www.w3.org/TR/WCAG21/#headings-and-labels)

**Comment by jaredwyee** [comment]
@m-sullivan7 what do you think about this one? I don't think the "relationship" of the principle number is particularly meaningful as to warrant inclusion in the header. Including it would also make the headers excessively long. I assume the reason to keep "Principle X" separate is to help focus on the actual text of the principle and allow for quicker visual scanning along the #1-9 list.

**Comment by m-sullivan7** [comment]
I agree. The only reason that we gave the principles numbers is because of their use in the principle rotator on the homepage. We wanted it to be clear to homepage visitors that the principle shown was one of many at first glance. It was design driving content. The principle numbers are in service of that, more than anything else. The numbers are definitely less important than the text of the principles.

I'm open to other design ideas on how to connect numbers to each principle if we want to look into it. I'm also OK if we decide we will not do this one.

**Comment by jaredwyee** [comment]
Thanks. Saying won't do for this.
